### ISSUE TITLE
should we place the dependency of React into peerDependencies ?

### ISSUE THREAD

**Issue by VitoVan** [i18next/react-i18next#273]
This is react-i18next:

```javascript
"dependencies": {
--
  | "hoist-non-react-statics": "1.2.0",
  | "prop-types": "^15.5.10",
  | "react": "^15.6.1"
  | },
```

https://github.com/i18next/react-i18next/blob/master/package.json#L27

this is react-dom:

```javascript
"peerDependencies": {
--
  | "react": "^16.0.0-alpha.13"
  | },
```

https://github.com/facebook/react/blob/master/packages/react-dom/package.json#L22

I am using React 16 in our project, the release[ v4.6.2 ](https://github.com/i18next/react-i18next/commit/58f32450d77edf07840f7b0d27938a5b5abd7aa4#diff-b9cfc7f2cdf78a7f4b91a753d10865a2)breaks entire project with the dependency of react 15.

Do we really need to lock React to v15 and write it in `dependencies`?

**Comment by jamuhl** [comment]
not sure how it happened but seems i added that accidently with a commit: https://github.com/i18next/react-i18next/commit/58f32450d77edf07840f7b0d27938a5b5abd7aa4

will remove it again and do a patch update...sorry.

**Comment by jamuhl** [comment]
published with react-i18next@4.6.3

thank you for reporting.
